build.gradle

```diff
@@ -65,9 +65,6 @@ jib {
 	from {
     	  image = 'openjdk:8-jre-alpine'
   	}
-  	to {
```

Comment:
if we run `./gradlew jibDockerBuild` then it will fail if we remove this.

Reply:
Also add `to.tags` as `unreleased` here such that when we run  ``` ./gradlew jibDockerBuild ``` then it will build unreleased image in local which we will reference in docker-compose.yml

Reply:
Please see comment below. Jib documentation states to.tags means additional tags. So if no initial tag is set in the image, then latest tag will automatically be added to the image. Additional tags do not prevent this behavior. 

I did find a workaround through, I can still have the image in the build.gradle file. But in the workflow still use to.image to overwrite the build.gradle image. Will commit that.